object_database/web/content/components/CodeEditor.js

```diff
@@ -42,7 +43,7 @@ class CodeEditor extends Component {
             this.editor.last_edit_millis = Date.now();
             this.editor.setTheme("ace/theme/textmate");
             this.editor.session.setMode("ace/mode/python");
-            this.editor.setAutoScrollEditorIntoView(true);
+            // this.editor.setAutoScrollEditorIntoView(true);
```

Comment:
does this mean that if I hit 'ctrl-G' it won't scroll that line into view? I don't want to alter the behavior of the component during regular editing - I just want to be able to control the edit state when I switch windows